generator/response.go

```diff
@@ -0,0 +1,69 @@
+package generator
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/go-swagno/swagno/components/parameter"
+	"github.com/go-swagno/swagno/utils"
+)
+
+type ResponseGenerator struct {
```

Comment:
if there's no state being managed do we really need a struct?

Reply:
just in case if we need just like DefinationGenerator